lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -53,6 +55,8 @@ sub register {
                 }
                 return $res[0];
             });
+        $app->helper('obs_rsync.get_batches'       => \&_get_batches);
```

Comment:
This helper seems to be only used once, shouldn't just get merged into the other helper that uses it?

Reply:
I planned to reuse it later, but if since you insist - I had to do it as part of this PR